### PR TITLE
Added options to not add the attendee list

### PR DESCRIPTION
**Pull request by rufuswilson**
Added option to select whether the attendee list should be copied across or not

### CODE REVIEW

**Comment by rufuswilson** [comment]
The problem I faced was that I invited 200 people to a duplicate event I originally didn't create and the duplicated event was cancelled when I deleted it.
There was a bit of confusion.

**Comment by jonas0b1011001** [comment]
So if you sync an event that is not originally yours, people on the attendee list receive an invitation/status update whenever you edit your personal copy of the event?

**Comment by rufuswilson** [comment]
An event is created in their calendar but they don't receive an invite by
email. However, if the event is deleted, they will receive a notification.

File: Helpers.gs [hunk 1]
```diff
@@ -126,7 +126,7 @@ function processEvent(event, calendarTz, calendarEventsMD5s){
     };
   }
   
-  if (event.hasProperty('attendee')){
+  if (copyAttendees && event.hasProperty('attendee')){
```
Comment (jonas0b1011001) [inline Helpers.gs]: You declared it as "AddAttendees", for uniformity please go with "addAttendees"

**Comment by jonas0b1011001** [comment]
Oh i see. That's bad, i never thought of that..sorry for the confusion it created for you.

File: Settings.gs [hunk 2]
```diff
@@ -14,6 +14,7 @@ var removeEventsFromCalendar = true;   // If you turn this to "true", any event
 var addAlerts = true;                  // Whether to add the ics/ical alerts as notifications on the Google Calendar events, this will override the standard reminders specified by the target calendar.
 var addOrganizerToTitle = false;       // Whether to prefix the event name with the event organiser for further clarity 
 var addCalToTitle = false;             // Whether to add the source calendar to title
+var AddAttendees = true;              // Whether to add the attendee list
```
Comment (jonas0b1011001) [inline Settings.gs]: We should probably add a remark that if set to true, it will create an event in the attendees' calendars

**Comment by rufuswilson** [comment]
I didn't know either until I received the first email asking if the meeting
was cancelled 😂

File: Settings.gs (hunk 2 above)
Comment (rufuswilson) [reply Settings.gs]: Sounds very sensible!

File: Settings.gs [hunk 3]
```diff
@@ -14,7 +14,7 @@ var removeEventsFromCalendar = true;   // If you turn this to "true", any event
 var addAlerts = true;                  // Whether to add the ics/ical alerts as notifications on the Google Calendar events, this will override the standard reminders specified by the target calendar.
 var addOrganizerToTitle = false;       // Whether to prefix the event name with the event organiser for further clarity 
 var addCalToTitle = false;             // Whether to add the source calendar to title
-var AddAttendees = true;              // Whether to add the attendee list
+var AddAttendees = true;              // Whether to add the attendee list. If true, duplicate events will be automatically added to the attendees' calendar.
```
Comment (jonas0b1011001) [inline Settings.gs]: The first 'A' needs to be lowercase.

**Pull request merged**